regression/ebmc/BDD/BDD_SVA1.desc

```diff
@@ -6,7 +6,7 @@ BDD_SVA1.sv
 ^\[top\.property\.p0\] always top\.my_bit: PROVED$
 ^\[top\.property\.p1\] always top\.my_bit: PROVED$
 ^\[top\.property\.p2\] always \(top\.counter == 3 \|-> \(nexttime top\.counter == 4\)\): FAILURE: property not supported by BDD engine$
-^\[top\.property\.p3\] always \(top\.counter == 3 \|=> top\.counter == 4\): PROVED$
+^\[top\.property\.p3\] always \(top\.counter == 3 \|=> top\.counter == 4\): FAILURE: property not supported by BDD engine$
```

Comment:
How does "netlist output" (as the commit message suggests) result in a change to property checking results?

Reply:
This goes as follows: the symbol table to setlist conversion now looks at the properties, and this uncovered a bug in the logic for what is supported by the net-list BMC.